regression/memory-analyzer/arrays/test.desc

```diff
@@ -0,0 +1,24 @@
+FUTURE
```

Comment:
Still `FUTURE`? If yes, please add comments (at the end of the file, after two `--` lines) to explain what's missing at the moment.

Reply:
Explanation added.